### ISSUE TITLE
Consolidation: Add Icons to Printable Version of Trial Session Information Screen

### ISSUE THREAD

**Issue by cholly75** [ustaxcourt/ef-cms#8396]
As a Court user, so that I can quickly identify consolidated cases associated with a particular trial session, I need the printable version of the Trial Session Information screen to indicate when a case is a lead or member case in a consolidated group.

This story will extend the consolidated iconography seen on the Trial Session Information screen to the printable version of same.

## Acceptance Criteria
* Cases calendared for a trial session that are Lead cases in a consolidated group are indicated with the "L" consolidated icon next to the docket number on the printable version of the Trial Session Information screen.
* Cases calendared for a trial session that are member cases in a consolidated group are indicated with the consolidated icon next to the docket number on the printable version of the Trial Session Information screen.
* Additional visual elements as detailed in UX comments below re: margins, etc.
* ADDED: Printable version should faithfully represent the screen version of the same data

## Tasks
- [x] Add icons to printable session copy PDF template
- [x] Reduce left/right hand margins to match TrialSessionWorkingCopy printable pdf
- [x] Add "n/a" in the session notes card when there is no input in that field

## Story Definition of Ready (updated on 12/23/22)
The following criteria must be met in order for the user story to be picked up by the Flexion development team.
The user story must: 
- [ ]  Is framed in business/user need, the value has been addressed.
- [ ] Includes acceptance criteria
- [ ] Has been refined
- [ ] Pre conditions have been satisfied. 
**Process:** 
Flexion developers and designers will test if the story meets acceptance criteria and test cases in Flexion dev and staging environments (“standard testing”). If additional acceptance criteria or testing scenarios are discovered while the story is in progress, a new story should be created, added to the backlog and prioritized by the product owner. 

## Definition of Done (Updated 5-19-22)
**Product Owner**
 - [ ]  Acceptance criteria have been met and validated on the Court's migration environment
 - [ ] Add scenario to testing document, if applicable (https://docs.google.com/spreadsheets/d/1FUHKC_YrT-PosaWD5gRVmsDzI1HS_U-8CyMIb-qX9EA/edit?usp=sharing)

**UX**
 - [ ] Business test scenarios have been refined to meet all acceptance criteria 
 - [ ] Usability has been validated
 - [ ] Wiki has been updated (if applicable) 
 - [ ] Story has been tested on a mobile device (for external users only)

**Engineering**
 - [x] Automated test scripts have been written, including visual tests for newly added PDFs
 - [x] Field level and page level validation errors (front-end and server-side) integrated and functioning
 - [x] Verify that language for docket record for internal users and external users is identical
 - [x] New screens have been added to pa11y scripts
 - [x] All new functionality verified to work with keyboard and macOS voiceover https://www.apple.com/voiceover/info/guide/_1124.html 
 - [x] READMEs, other appropriate docs, JSDocs and swagger/APIs fully updated
 - [x] UI should be touch optimized and responsive for external only (functions on supported mobile devices and optimized for screen sizes as required)
 - [x] Interactors should validate entities before calling persistence methods
 - [x] Code refactored for clarity and to remove any known technical debt
 - [x] Acceptance criteria for the story has been met
 - [x] If there are special instructions in order to deploy into the next environment, add them as a comment in the story
 - [x] Reviewed by UX on a deployed environment.
 - [x] Reviewed by PO on a deployed environment. Can be deployed to the Court's `test` environment if prod-like data is required. Otherwise deployed to any `experimental` environment.
 - [x] Deployed to the Court's `staging` environment.

**Comment by swongCO** [comment]
Pre-refinement:
Use Session Copy page pdf as reference
- reduce side margins

**Comment by katiecissell** [comment]
UX Notes:
Implementation should follow the pattern set by the Session Copy page PDF.
- Same patterns for the icons
- reduce the left and right margins of the trial session pdf to match the session copy pdf's margins (to give some more room in the table)
- add "n/a" in the session notes card when there is no input in that field

What session copy print out looks like (as a reminder):

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBak9CIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--935ccd78dd76cebb210224ae06c94770bde5db0c/image.png)

**Comment by ttlenard** [comment]
## Test Case

#### 1) Court User accesses a Trial Session that has consolidated cases; Prints the Trial Session Information page; printable version has Consolidated Icons present.

- Log in as any Court User
- Click on the Trial Sessions Header
- Click on the City, State link of an open session that has consolidated cases scheduled for trial
- Review the display of the consolidated Cases and the ordering on the Session information page.
- Click on the Print link
- Review the screen with the document generated for printing

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Lead Case icon is present next to the Lead Case
- [x] **NEW FUNCTIONALITY** - The member Case icon is present next to all of the member cases in the group
- [x] **NEW FUNCTIONALITY** - The member cases are indented under the lead case
- [x] **NEW FUNCTIONALITY** - Margins are smaller per UX feedback
- [x] **NEW FUNCTIONALITY** - If there are no session notes, n/a is displayed
- [x] The cases displayed on the printed report are in the same order as displayed on the Trial Session information page.

*Repeat this test with various Court Users

**Comment by swongCO** [comment]
UX test feedback:
- counsel are not displaying on the pdf
- vertically center icons to docket numbers
- add more space to the bottom of each row for note taking purposes
- see screenshots for examples from Session Copy pdf:

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBampxIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--e278f57a80f1c749cc1fb657bfd37e663d88c9ad/Screenshot%202023-09-28%20at%209.36.13%20AM.png)

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBam5xIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--81a9272a12b252030e5253c08ed12b924d49e739/Screenshot%202023-09-28%20at%209.35.55%20AM.png)

**Comment by swongCO** [comment]
UX test feedback

Tested with Denver, Colorado 01/08/24 session:
- one consolidated group is not grouped together in the pdf but is in the Session Info page (there's another group that is grouped together in both):

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbDd3IiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--74679bb55f8b082b24d7450b8e0163e3346bf8c1/Screenshot%202023-09-29%20at%209.54.43%20AM.png)

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbUR3IiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--ce0b2144950a3c6c0a9b2f4bfddd5c160b5e3b42/Screenshot%202023-09-29%20at%209.58.12%20AM.png)

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbUh3IiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--9232401d991a538dc5cde2f7f5f2fdf10d450bf5/Screenshot%202023-09-29%20at%209.58.27%20AM.png)

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbUx3IiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--657728c11cfd4ff9aa54fe8f2e04595320d72635/Screenshot%202023-09-29%20at%209.58.45%20AM.png)

**Comment by ttlenard** [comment]
Testing Feedback:
@rachelschneiderman @Absolutestunna @jasonfu9966 

I found that the Printed version and the UI version don't quite match up if the lead case in a consolidated group is removed from the trial, but the member cases remain. On the UI, I'm pretty sure that the "rule" is When lead case is removed from trial, member cases are no longer indented or grouped together.

Here is a screen grab showing this:
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbnYzIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--851a14344781067f276b3536ae3539d816655620/image.png)

But then on the Printed version of this Trial Session page, the cases do not appear in the same order.  Looks like the member cases are still indented and grouped, without the lead case. In this scenario, it looks like this group is part of the consolidated group above it.
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbjMzIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--859c278e704a1329ccb35129aa164d5cb19e9094/image.png)

A similar behavior is occurring when you print the Session Copy page (not part of this story). I talked with Chris about this, and I think we may address this in a different story to fix this on the printed versions of both pages at once.

Let me know if you have any feedback on this and if this is the route that we should take to ensure that what the user sees on the UI is the same in the Printed Version.
